internal/builders/rust/testdata/proj/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "proj"
+version = "0.1.0"
+edition = "2021"
```

Comment:
nit and not related to this PR: you may want to track 2024 edition that's coming soon (available at nightly).

Reply:
we can add more tests later, I figured it shouldn't change the implementation we have so far though.

Reply:
no no, should be okey 👌